src/mergePath.js

```diff
@@ -0,0 +1,27 @@
+import { curry, over, lensPath, merge, __ } from 'ramda';
+
+/**
+ * Merges user provided data (subject) with some object (target)
+ * inside other object under corresponding path.
+ *
+ * @func mergePath
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.18.0|v1.18.0}
+ * @category Object
+ * @sig [k] -> {a} -> {k: {a}} -> {k: {a}}
+ * @see {@link RA.mergeProp|mergeProp}
+ * @param {!Array} path The path to object to merge with
+ * @param {!Object} subj The object which will be merged with object in path
+ * @param {!Object} obj The object which has target object under corresponding path
+ * @return {!Object} The new version of object
+ * @example
+ *
+ * RA.mergePath(
```

Comment:
This example is IMHO too complicated due to the naming of the propertie . Can we simplify it some more primitive usecase ?

```javascript
const obj = { outer: { inner: { foo: 2 } } };

RA.mergePath(['outer', 'inner'], { foo: 3, bar: 4 }, obj); //=> { outer: { inner: { foo: 3, bar: 4 } }
```
Note: use spaces to indent `{`, `}` chars. Airbnb codestyle...